src/index.html

```diff
@@ -4,9 +4,381 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link 
+      rel="stylesheet" 
+      href="./styles/main.scss"
+    >
+    
+    <link 
+      rel="preconnect" 
+      href="https://fonts.googleapis.com"
+    >
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com" 
+    >
+    <link 
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" 
+      rel="stylesheet"
+      crossorigin="anonymous"
+    >
   </head>
+
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <img
```

Comment:
There is a common HTML structure for the logo block:

```
<div class="header__logo">
  <a href="..." class="...">
    <img 
      src="..."
      alt="..."
      class="..."
     >
  </a>
</div>
```

The link should direct to the homepage of your website.
We also can add width and height attributes for the logo image in the header to avoid content flickering after CSSOM is built.
To know more about this you should read [the article about the critical rendering path and how the browser parses HTML files.](https://developer.mozilla.org/en-US/docs/Web/Performance/Critical_rendering_path)